src/tests/join/join-tests.cu

```diff
@@ -644,3 +613,50 @@ TYPED_TEST(JoinTest, RightColumnsBigger)
     EXPECT_EQ(reference_result[i], gdf_result[i]);
   }
 }
+
+// TODO Add test for inputs with zero rows
```

Comment:
Should add tests to check the zero matching rate (either no matches, or an empty dataset?)

Reply:
Done.